index.bs

```diff
@@ -147,11 +145,80 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
 
 <p>In [[!ISOBMFF]] or [[!HEIF]] files, if an [=AV1 Alpha Image Item=] (respectively. an [=AV1 Alpha Image Sequence=]) is used, the <code>aux_type</code> field of the <code>AuxiliaryTypeProperty</code> (respectively <code>AuxiliaryTypeInfoBox</code>) SHALL be set to <code>urn:mpeg:mpegB:cicp:systems:auxiliary:alpha</code>, as defined in [[!MIAF]].</p>
 
-<h2 id="AVIF-baseline-profile">MIAF AV1 Baseline Profile</h2>
+  <h2 id="brands-and-file-extensions">Brands and file extensions</h2>
+
+  <h3 id="AVIF-general-brand">AVIF General Brand</h3>
+
+Files conformant with the brand-independent restrictions in this document shall include the brand
+<dfn value="" export="" for="AVIF General brand">avif</dfn> in the [=compatible_brands=] field of the [=FileTypeBox=].
+  Files shall also carry a compatible brand to identify the AVIF profile, if any, with which the file complies.
+
+  <h3 id="image-and-image-collection-files">AV1 image and image collection files</h3>
+  The file extension ".avif" and the MIME Content Type "image/avif" should be used for AVIF files that satisfy the following requirements:
```

Comment:
Do we really want a separate MIME type? Can't we use HEIF with codecs = av1?

Reply:
The "codecs" parameter isn't universally recognized and isn't even explicitly mentioned as something that may occur on the image/heif MIME type, so, yes, I think it will be very helpful to have a separate MIME type and file extension. (Basically, I have seen the "codecs" parameter used in MPEG-DASH but it is not used much outside of that feature area.)

Reply:
> The "codecs" parameter isn't universally recognized

I think it is for ISOBMFF-based files.

> isn't even explicitly mentioned as something that may occur on the image/heif MIME type

Annex D and Annex E both define the 'codecs' parameter. It is also in the format IANA registry: https://www.iana.org/assignments/media-types/image/heif
Note that there are 3 other sub-parameters: profiles, itemtypes and lhevcptl. The first two could make sense for AVIF too.

Reply:
When I said the codecs parameter is not universally recognized, it was not a conjecture. I am responsible for media file MIME types in Windows and on Xbox One, so I should know... If we want to design a highly interoperable file format, then we need to have a separate file extension for this file format.

Reply:
I'm not opposed to defining a file extension. I'm just saying it won't be the only way of delivering AVIF files.